### PR TITLE
CRS-1468 Make editor compatible with Rails 4.2.x

### PR DESCRIPTION
**Pull request by lluchez**
Closes https://snapsheettech.atlassian.net/browse/CRS-1468

## What/Why/How

- Fork WYSIWYG editor from `kivanio/activeadmin-wysihtml5`
- Drop support to `dragonfly` which isn't compatible with Rails `4.2.x`
- Make sure editor is still properly working even with the change of version of `activeadmin` (from `1.0.0.pre` to `1.0.0` which includes some changes with the CSS mixins).

## Testing

Can be tested [here](https://github.com/bodyshopbidsdotcom/guidelines/pull/207). It's currently using the [same changes](https://github.com/kivanio/activeadmin-wysihtml5/compare/master...lluchez:master) but done on [another repo](https://github.com/lluchez/activeadmin-wysihtml5)

### CODE REVIEW

File: README.html [hunk 1]
```diff
@@ -0,0 +1,1117 @@
+<!DOCTYPE html><html><head><meta charset="utf-8"><meta name="viewport" content="width=device-width, initial-scale=1"><style>body {
```
Comment (dangarcia) [inline README.html]: what was the purpose of adding this file?

Comment (lluchez) [reply README.html]: Hmm.... I'm not sure how it got in. I'll remove it.

File: README.md [hunk 2]
```diff
@@ -20,17 +16,19 @@ interface using [wysihtml5](https://github.com/xing/wysihtml5).
 ```ruby
 # Gemfile
 
-gem 'activeadmin-dragonfly', github: 'stefanoverna/activeadmin-dragonfly'
+# gem 'activeadmin-dragonfly', github: 'stefanoverna/activeadmin-dragonfly'
```
Comment (santi-h) [inline README.md:19]: commenting out in the `README.md`? why not just remove the line?

**Review by santi-h — APPROVED** [review]
one non blocking comment

**Review by dangarcia — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (lluchez) [reply README.md:19]: We are dropping support to document upload but I feel like we could eventually re-add it with more work.
I kind of wanted to keep that here to find it easily, even if we look at it back in a few months.

**Pull request merged**